documentation/types.ts

```diff
@@ -41,10 +42,16 @@ export interface CompiledExample {
 
 export interface LinkData {
   type: UrlType;
-  url: string;
+  url?: string;
 }
 
 export enum UrlType {
+  SamplePage = 'SamplePage',
   SdkReferencePath = 'SdkReferencePath',
   Web = 'Web',
 }
+
+export enum ExampleCategory {
+  Topic = 'topic',
```

Comment:
Nit, the enum value convention is inconsistent between this and the UrlType enum right above it. Maybe capitalize these?

Reply:
Fixed.